recipes/snowflake/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "snowflake-connector-python" %}
+{% set version = "1.6.10" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ef4cdac45402561dd2bf16e217cd7915b9fc2fd2444ed6c2715b8a6e90a3afbf
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+    build:
```

Comment:
please, change that to ``host`` and add ``pip`` as a dependency

Reply:
Thanks @marcelotrevisani ... I'm still a bit wrapped around the axle with the conda-build 2 vs conda-build 3 syntax here 😸